.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,41 @@
+name: Python 3 testing
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 1
+      matrix:
+        python-version: [3.6]  # [2.7, 3.5, 3.6,, 3.7]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt || true
+    - name: Compile V8
+      run: |
+        python ./configure.py
+        make -j2 -C out V=1 v8
```

Comment:
This can be dropped. We only had it on Travis for a while to get around the 50 min job time limit and have since dropped it from Travis in favour of imposing our own timeout on the Compile Node.js step.